bindata/etcd/pod.yaml

```diff
@@ -10,6 +10,46 @@ metadata:
     revision: "REVISION"
 spec:
   initContainers:
+    - name: wait-for-port
+      image: ${IMAGE}
+      imagePullPolicy: IfNotPresent
+      terminationMessagePolicy: FallbackToLogsOnError
+      securityContext:
+        privileged: true
+      command:
+        - /bin/sh
+        - -c
+        - |
+          #!/bin/sh
+          set -euo pipefail
+
+          echo -n "Waiting for ports 2379, 2380 and 9978 to be released."
+          while [ -n "$(lsof -ni :2379)$(lsof -ni :2380)$(lsof -ni :9978)" ]; do
```

Comment:
/hold

the check needs to be in the same container as listen - if a container restarts this check wouldn't be executed again

Reply:
I restored the check during startup, and also duplicated the check as a precondition in the defrag container to help ensure we don't perform the operation against a live(ish) member.

Thanks!